userguide/tasks.rst

```diff
@@ -283,74 +283,98 @@ task.
+   |                                  |                             |                                                                                           |
+   +----------------------------------+-----------------------------+-------------------------------------------------------------------------------------------+
+   | Times                            | checkbox                    | Set to preserve the modification times of files.                                          |
+   |                                  |                             |                                                                                           |
+   +----------------------------------+-----------------------------+-------------------------------------------------------------------------------------------+
+   | Compress                         | checkbox                    | Set to reduce the size of the data to transmit. Recommended for slow connections.         |
+   |                                  |                             |                                                                                           |
+   +----------------------------------+-----------------------------+-------------------------------------------------------------------------------------------+
+   | Archive                          | checkbox                    | When set, rsync is run recursively, preserving symlinks, permissions, modification times, |
+   |                                  |                             | group, and special files. When run as root, owner, device files, and special files are    |
+   |                                  |                             | also preserved. Equivalent to :samp:`rsync -rlptgoD`.                                     |
+   |                                  |                             |                                                                                           |
+   +----------------------------------+-----------------------------+-------------------------------------------------------------------------------------------+
+   | Delete                           | checkbox                    | Set to delete files in the destination directory that do not exist in the source          |
+   |                                  |                             | directory.                                                                                |
+   |                                  |                             |                                                                                           |
+   +----------------------------------+-----------------------------+-------------------------------------------------------------------------------------------+
+   | Quiet                            | checkbox                    | Set to suppress informational messages from the remote server.                            |
```

Comment:
The change to this field description needs to be ported over to your PR: https://github.com/freenas/freenas-docs/commit/3553f342c5d3a7486bfe8afbba97fb0cf23eaa21